### PR TITLE
Idan s refactoring popup.js and some improvements

### PR DESCRIPTION
**Pull request by idan2468**
- Refactor the popup.js
- Resolved the issue of running scripts file with `browser.tabs.executeScript`
  so now it's running without the minified text:
  `browser.tabs.executeScript({file: '/content_scripts/courseRemover.js'}).catch(reportError)`
see issue #121 for details
image

- Changed extension popup behavior see issue #122 for more details

### CODE REVIEW

File: content_scripts/courseScheduale.js [hunk 1]
```diff
@@ -34,17 +38,18 @@ const getTemplate = (minTime, maxTime) => {
...
     <table class="scheduale-table">
         <thead>
             <tr>
                 <th style="width: 3%;"></th>
-                <th style="width: 19%;"> א </th>
-                <th style="width: 19%;"> ב </th>
-                <th style="width: 19%;"> ג </th>
-                <th style="width: 19%;"> ד </th>
-                <th style="width: 19%;"> ה </th>
+                <th style="width: 19%;"> ${isHebrew ? 'א' : 'Sunday'} </th>
```
Comment (norbit8) [inline content_scripts/courseScheduale.js]: Are you sure this file should be in this PR? it's already in #116

File: popup/popup.js [hunk 2]
```diff
@@ -12,207 +10,148 @@ async function listenForClicks() {
...
         async function actionToScript(id) {
             switch (id) {
-                // TODO: maybe implement your own css or take the class you need from bootstrap for buttons instead of injecting bootstrap css to the page
                 case "remove-courses":
-                    browser.tabs.executeScript({code: cr}).catch(
-                        reportError
-                    );
-                    courseRemoverSwitch().catch(e => console.log(e));
-                    return;
+                    browser.tabs.executeScript({file: '/content_scripts/courseRemover.js'}).catch(reportError)
```
Comment (norbit8) [inline popup/popup.js:17]: Does it work in chrome?

**Review by norbit8 — CHANGES_REQUESTED** [review]
Seems like you've mixed some of the PR's changes from #116 to here.
Also, are you sure this solution works for chrome?

**Comment by idan2468** [comment]
> Seems like you've mixed some of the PR's changes from #116 to here.
> Also, are you sure this solution works for chrome?

Yeah it's possible, and yes I've checked it on both. You can see the image I've added explains that you need to put the absolute path (linux style)

**Comment by norbit8** [comment]
> > Seems like you've mixed some of the PR's changes from #116 to here.
> > Also, are you sure this solution works for chrome?
> 
> Yeah it's possible, and yes I've checked it on both. You can see the image I've added explains that you need to put the absolute path (linux style)

Alright, so just remove the redundant files and this PR will be approved 💯

**Review by norbit8 — APPROVED** [review]

**Pull request merged**